helm/bundle/manifests/minio-directpv-operator-rhmp.clusterserviceversion.yaml

```diff
@@ -318,14 +318,15 @@ spec:
   - supported: true
     type: AllNamespaces
   keywords:
-  - directpv
+  - minio-directpv-operator-rhmp
   links:
-  - name: Directpv Operator
-    url: https://directpv-operator.domain
+  - name: Minio Directpv Operator Rhmp
```

Comment:
We would need to style DirectPV as `DirectPV` like ` - name: Minio DirectPV Operator RHMP`